src/main/clojure/clara/rules/engine.cljc

```diff
@@ -1702,6 +1702,18 @@
       (filter (fn [fact] (join-filter-fn token fact fact-bindings {}))
               unfiltered-facts))))
 
+;; This lives here as it is both close to the node that it represents, and is accessible to both clj and cljs
+(def node-type->abbreviated-type
```

Comment:
Curious why this ended up being in this ns instead of compiler?

Reply:
@EthanEChristian answered me offline, only this ns can be reached from cljs currently. That's fine for now.